Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -1759,18 +1782,20 @@ void EmbeddedFrag::openAngles(const double *dmat, unsigned int aid1,
   PRECONDITION(dmat, "");
   unsigned int deg1 = getHeavyDegree(dp_mol->getAtomWithIdx(aid1));
   unsigned int deg2 = getHeavyDegree(dp_mol->getAtomWithIdx(aid2));
-  if ((deg1 > 1) && (deg2 > 1)) {
+  bool fixed1 = d_eatoms[aid1].df_fixed;
+  bool fixed2 = d_eatoms[aid2].df_fixed;
+  if ((deg1 > 1 || fixed1) && (deg2 > 1 || fixed2)) {
     return;
   }
   unsigned int aidA;
   unsigned int aidB;
   RDKit::ROMol::ADJ_ITER nbrIdx, endNbrs;
   int type = 0;
-  if ((deg1 == 1) && (deg2 == 1)) {
+  if ((deg1 == 1 && !fixed1) && (deg2 == 1 && !fixed2)) {
     aidA = _findDeg1Neighbor(dp_mol, aid1);
     aidB = _findDeg1Neighbor(dp_mol, aid2);
     type = 1;
-  } else if ((deg1 == 1) && (deg2 > 1)) {
+  } else if ((deg1 == 1 && !fixed1) && (deg2 > 1 || fixed1)) {
```

Comment:
yeah, that's a typo. Fix coming in a minute or two (once the tests finish running)